### ISSUE TITLE
Installing on raspberry pi fails

### ISSUE THREAD

**Issue by Le0Developer** [alisaifee/coredis#42]
(dont mind me pinning the version to 3.2.0, doesn't work on latest either, 3.1 works)
```py
pi@rasp-3b-01:~/backend $ pip3.10 install coredis==3.2.0
Defaulting to user installation because normal site-packages is not writeable
Collecting coredis==3.2.0
  Downloading coredis-3.2.0.tar.gz (150 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 150.5/150.5 KB 1.2 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Requirement already satisfied: deprecated>=1.2 in /home/pi/.local/lib/python3.10/site-packages (from coredis==3.2.0) (1.2.13)
Requirement already satisfied: typing_extensions>=4.1.1 in /home/pi/.local/lib/python3.10/site-packages (from coredis==3.2.0) (4.1.1)
Requirement already satisfied: packaging<22,>=21 in /home/pi/.local/lib/python3.10/site-packages (from coredis==3.2.0) (21.3)
Requirement already satisfied: wrapt<2,>=1.1.0 in /home/pi/.local/lib/python3.10/site-packages (from coredis==3.2.0) (1.14.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/pi/.local/lib/python3.10/site-packages (from packaging<22,>=21->coredis==3.2.0) (3.0.7)
Building wheels for collected packages: coredis
  Building wheel for coredis (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [24 lines of output]
      running bdist_wheel
      running build
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-m9vbv9fv/coredis_bbd8208755624fe0a8cf638f64d95289/setup.py", line 128, in <module>
          setup(
        File "/usr/local/lib/python3.10/site-packages/setuptools/__init__.py", line 153, in setup
          return distutils.core.setup(**attrs)
        File "/usr/local/lib/python3.10/distutils/core.py", line 148, in setup
          dist.run_commands()
        File "/usr/local/lib/python3.10/distutils/dist.py", line 966, in run_commands
          self.run_command(cmd)
        File "/usr/local/lib/python3.10/distutils/dist.py", line 985, in run_command
          cmd_obj.run()
        File "/home/pi/.local/lib/python3.10/site-packages/wheel/bdist_wheel.py", line 299, in run
          self.run_command('build')
        File "/usr/local/lib/python3.10/distutils/cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "/usr/local/lib/python3.10/distutils/dist.py", line 985, in run_command
          cmd_obj.run()
        File "/tmp/pip-install-m9vbv9fv/coredis_bbd8208755624fe0a8cf638f64d95289/setup.py", line 36, in run
          import scripts.code_gen
      ModuleNotFoundError: No module named 'scripts'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for coredis
  Running setup.py clean for coredis
Failed to build coredis
Installing collected packages: coredis
  Running setup.py install for coredis ... error
  error: subprocess-exited-with-error
  
  × Running setup.py install for coredis did not run successfully.
  │ exit code: 1
  ╰─> [26 lines of output]
      running install
      running build
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/tmp/pip-install-m9vbv9fv/coredis_bbd8208755624fe0a8cf638f64d95289/setup.py", line 128, in <module>
          setup(
        File "/usr/local/lib/python3.10/site-packages/setuptools/__init__.py", line 153, in setup
          return distutils.core.setup(**attrs)
        File "/usr/local/lib/python3.10/distutils/core.py", line 148, in setup
          dist.run_commands()
        File "/usr/local/lib/python3.10/distutils/dist.py", line 966, in run_commands
          self.run_command(cmd)
        File "/usr/local/lib/python3.10/distutils/dist.py", line 985, in run_command
          cmd_obj.run()
        File "/usr/local/lib/python3.10/site-packages/setuptools/command/install.py", line 61, in run
          return orig.install.run(self)
        File "/usr/local/lib/python3.10/distutils/command/install.py", line 568, in run
          self.run_command('build')
        File "/usr/local/lib/python3.10/distutils/cmd.py", line 313, in run_command
          self.distribution.run_command(command)
        File "/usr/local/lib/python3.10/distutils/dist.py", line 985, in run_command
          cmd_obj.run()
        File "/tmp/pip-install-m9vbv9fv/coredis_bbd8208755624fe0a8cf638f64d95289/setup.py", line 36, in run
          import scripts.code_gen
      ModuleNotFoundError: No module named 'scripts'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> coredis

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

**Comment by alisaifee** [comment]
Ouch! I totally didn't account for that installation workflow.
Could you try the latest on master (469a6584b6b6bc4d160cda72d70aafb473dc51ac)

If that works - I'll cut a release tomorrow.

**Comment by Le0Developer** [comment]
Works on master!

**Comment by alisaifee** [comment]
Fix is available in [3.4.0](https://coredis.readthedocs.io/en/stable/release_notes.html#v3-4-0).
